README.md

```diff
@@ -27,12 +27,12 @@ $ make # You can find the binary file in the `_build` dir if all goes good
 ```
 ### Overview
 ![image](docs/images/overview.png)
-For the storage, the ETCD is used as the default storage now. Welcome to contribute other storages like MySQL, Redis, Consul and so on. And what you need to do is to implement the [persistence interface](https://github.com/apache/kvrocks-controller/blob/unstable/storage/persistence/persistence.go).
+For the storage, the ETCD is used as the default storage now. Welcome to contribute other storages like MySQL, Redis, Consul and so on. And what you need to do is to implement the [persistence interface](https ://github.com/apache/kvrocks-controller/blob/unstable/store/engine/engine.go).
```

Comment:
Additionally, we can change `persistance` => `Engine`.

Reply:
`persistence interface` => `Engine` 
Maybe it is more appropriate, because Engine is not an interface

Reply:
@caicancai Why is `Engine` not an interface?
![image](https://github.com/user-attachments/assets/0a5b4062-45b5-494a-a099-923508588188)

Reply:
@torwig Sorry, I didn't click on the file inside, I thought it was the file name, it's still building. 
Thanks for the correction

Reply:
done